paddle/framework/CMakeLists.txt

```diff
@@ -55,5 +55,6 @@ cc_library(paddle_pybind SHARED
     recurrent_op
     uniform_random_op
     gaussian_random_op
+    lookup_table_op
```

Comment:
It seems that the solution is to move target pybind into a third package, say, `paddle/pybind`, in parallel with `paddle/framework` and `paddle/operators`.